src/GoodsList.tsx

```diff
@@ -1,14 +1,22 @@
+import classNames from 'classnames';
 import React from 'react';
 import { Good } from './types/Good';
 
 type Props = {
   goods: Good[]
+  red: boolean
 };
 
-export const GoodsList: React.FC<Props> = ({ goods }) => (
+export const GoodsList: React.FC<Props> = ({ goods, red }) => (
   <ul className="has-text-centered">
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <li
+        key={good.id}
+        data-cy="good"
+        className={classNames('has-text-black', {
```

Comment:
Unfortunately, it's still not red in the demo:

![image](https://user-images.githubusercontent.com/86923089/189963596-66e1d298-64fa-44b1-85f9-8a539b662387.png)

And that's not only about red goods:
![image](https://user-images.githubusercontent.com/86923089/189963773-d19c5e15-1dd2-4f79-ad25-a08077853f9e.png)

Nice Bulma styles! but with those colors would be easier to just use `color` property from `good` object